### ISSUE TITLE
Can't use in a project with `#![deny(missing_docs)]`

### ISSUE THREAD

**Issue by marshallpierce** [la10736/rstest#106]
Trying to use the example test code in the readme with 
```
#![deny(missing_docs)]

#[cfg(test)]
use rstest_reuse;
```
in the top level of the crate produces (with `RUSTFLAGS="-Z macro-backtrace"` and the nightly compiler):

```
error: missing documentation for macro
   --> src/lib.rs:16:5
    |
16  |     #[template]
    |     ^^^^^^^^^^^ in this macro invocation
    |
   ::: /Users/mbp/.cargo/registry/src/github.com-1ecc6299db9ec823/rstest_reuse-0.1.0/src/lib.rs:179:1
    |
179 | pub fn template(_args: proc_macro::TokenStream, input: proc_macro::TokenStream) -> TokenStream {
    | ---------------------------------------------------------------------------------------------- in this expansion of `#[template]`
    |
note: the lint level is defined here
   --> src/lib.rs:3:9
    |
3   | #![deny(missing_docs)]
    |         ^^^^^^^^^^^^
```

Perhaps adding `/// Generated` to the generated code is all that's needed?

Edit: also, adding `#![allow(missing_docs)]` in the module containing the tests doesn't help.

**Comment by la10736** [comment]
Thanks to reporting it. I'll take a look this weekend.

Maybe I'll just provide a default doc for generated macro.

**Comment by la10736** [comment]
Should be fixed with 0.1.1 version released on crates.io

**Comment by marshallpierce** [comment]
Great, thanks!
